lib/Doctrine/Annotations/Annotation.php

```diff
@@ -1,62 +0,0 @@
-<?php
-
-namespace Doctrine\Annotations;
-
-/**
- * Annotations class.
- *
- * @author Benjamin Eberlei <kontakt@beberlei.de>
- * @author Guilherme Blanco <guilhermeblanco@hotmail.com>
- * @author Jonathan Wage <jonwage@gmail.com>
- * @author Roman Borschel <roman@code-factory.org>
- */
-class Annotation
```

Comment:
I don't think the base class should be removed, as it makes the BC break muvh bigger than needed: a deprecation would probably be better

Reply:
I was rather thinking about deprecating it in 1.7 and going with less burdens in 2.0. Mostly because this class is rarely used out there. But I may be wrong, GitHub search is not very helpful in finding usages.

Reply:
Deprecating it in 1.7 would be the first step before dropping it from 2.0. We can then see what people say and either decide to revert the deprecation or go along with the removal.